interface/forms_admin/forms_admin.php

```diff
@@ -37,103 +42,107 @@
-<input type=submit name=update value='<?php xl('update','e'); ?>'><br>
+<i><?php echo xlt('click here to update priority, category, nickname and access control settings'); ?></i>
+<input type='submit' name='update' value='<?php echo xla('update'); ?>'><br>
 <table border=0 cellpadding=1 cellspacing=2 width="500">
 	<tr>
 		<td> </td>
 		<td> </td>
 		<td> </td>
 		<td> </td>
 		<td> </td>
-		<td><?php xl('Priority ','e'); ?></td>
-		<td><?php xl('Category ','e'); ?></td>
-		<td><?php xl('Nickname','e'); ?></td>
+		<td><?php echo xlt('Priority'); ?> </td>
+		<td><?php echo xlt('Category'); ?> </td>
+		<td><?php echo xlt('Nickname'); ?> </td>
+		<td><?php echo xlt('Access Control'); ?></td>
 	</tr>
 <?php
 $color="#CCCCCC";
 if ($bigdata != false)
 foreach($bigdata as $registry)
 {
-	$priority_category = sqlQuery("select priority, category, nickname from registry where id=".$registry['id']);
-	?>
-	<tr>
-		<td bgcolor="<?php echo $color?>" width="2%">
-			<span class=text><?php echo $registry['id'];?></span>
-		</td>
-		<td bgcolor="<?php echo $color?>" width="30%">
-			<span class=bold><?php echo xl_form_title($registry['name']); ?></span>
-		</td>
-		<?php
-			if ($registry['sql_run'] == 0)
-				echo "<td bgcolor='$color' width='10%'><span class='text'>".xl('registered')."</span>";
-			elseif ($registry['state'] == "0")
-				echo "<td bgcolor='#FFCCCC' width='10%'><a class=link_submit href='./forms_admin.php?id={$registry['id']}&method=enable'>".xl('disabled')."</a>";
-			else
-				echo "<td bgcolor='#CCFFCC' width='10%'><a class=link_submit href='./forms_admin.php?id={$registry['id']}&method=disable'>".xl('enabled')."</a>";
-		?></td>
-		<td bgcolor="<?php $color?>" width="10%">
-			<span class=text><?php
-
-			if ($registry['unpackaged'])
-				echo xl('PHP extracted','e');
-			else
-				echo xl('PHP compressed','e');
-
-			?></span>
-		</td>
-		<td bgcolor="<?php echo $color?>" width="10%">
-			<?php
-			if ($registry['sql_run'])
-				echo "<span class=text>".xl('DB installed')."</span>";
-			else
-				echo "<a class=link_submit href='./forms_admin.php?id={$registry['id']}&method=install_db'>".xl('install DB')."</a>";
-			?>
-		</td>
-		<?php
-			echo "<td><input type=text size=4 name=priority_".$registry['id']." value='".$priority_category['priority']."'></td>";
-			echo "<td><input type=text size=8 name=category_".$registry['id']." value='".$priority_category['category']."'></td>";
-			echo "<td><input type=text size=8 name=nickname_".$registry['id']." value='".$priority_category['nickname']."'></td>";
-		?>
-	</tr>
-	<?php
-	if ($color=="#CCCCCC")
-		$color="#999999";
-	else
-		$color="#CCCCCC";
+  $priority_category = sqlQuery("select priority, category, nickname, aco_spec from registry where id = ?",
+    array($registry['id']));
+?>
+  <tr>
+    <td bgcolor="<?php echo $color; ?>" width="2%">
+      <span class='text'><?php echo $registry['id']; ?></span>
+    </td>
+    <td bgcolor="<?php echo $color; ?>" width="30%">
```

Comment:
attr()
(I realize this is a bit on the anal side, but never know if $color will become a option from user selection in the future :) )